docs/demos/tutorials/02_Exploratory_analysis.ipynb

```diff
@@ -2619,23 +2619,23 @@
   {
    "attachments": {},
    "cell_type": "markdown",
-   "id": "1f37cb1e",
+   "id": "c537f065-f61f-402a-ad2c-a8575b6207ab",
    "metadata": {},
    "source": [
-    "## Next steps\n",
+    "!!! note \"Further Reading\"\n",
     "\n",
-    "At this point, we have begin to develop a strong understanding of our data.  It's time to move on to estimating a linkage model\n"
+    "    :material-tools: For more on exploratory analysis tools in Splink, please refer to the [Exploratory Analysis API documentation](../../linkerexp.md)."
    ]
   },
   {
    "attachments": {},
    "cell_type": "markdown",
-   "id": "c537f065-f61f-402a-ad2c-a8575b6207ab",
+   "id": "1f37cb1e",
    "metadata": {},
    "source": [
-    "## Further reading\n",
+    "## Next steps\n",
     "\n",
-    "You can find the documentation for the exploratory analysis tools in Splink [here](https://moj-analytical-services.github.io/splink/linkerexp.html)"
+    "At this point, we have begin to develop a strong understanding of our data.  It's time to move on to estimating a linkage model\n"
```

Comment:
https://github.com/moj-analytical-services/splink/pull/1449#issuecomment-1666827466

Reply:
Is there a reason we only seem to have a "Next Steps" section for explo analysis?

Reply:
Nope, that was a mistake. Have added now